### ISSUE TITLE
700987 has an LC call # but no keymap_struct

### ISSUE THREAD

**Issue by banukutlu** [psu-libraries/psulib_traject#461]
[700987](https://catalog.libraries.psu.edu/catalog/700987) has an LC call number but has no keymap structure, no button, and is not showing up in Browse.

**Comment by banukutlu** [comment]
@ruthtillman We looked into this [700987](https://catalog.libraries.psu.edu/catalog/700987) record with @ajkiessl  and @whereismyjetpack  and what we found out happening is that this record has acces_facet as 

```
[
"In the Library",
"Online"
],
```

and our browse logic checks if access_facet includes 'Online' to reject the call number as non browsable.

But this record has access_facet facet online because of hathi access not the holding location being online.
We are thinking to change this check in the browse as if includes 'In the Library' then there is a browsable call number and it should just return it. We were wondering if our thinking sound right and make sense to you?
